crates/trippy-tui/locales/app.yml

```diff
@@ -6,730 +6,835 @@ trippy:
 udp:
   en: "udp"
   fr: "udp"
   tr: "udp"
   it: "udp"
   pt: "udp"
   zh: "udp"
+  ar: "udp"
 tcp:
   en: "tcp"
   fr: "tcp"
   tr: "tcp"
   it: "tcp"
   pt: "tcp"
   zh: "tcp"
+  ar: "tcp"
 status_failures:
   en: "%{failure_count} of %{total_probes} (%{failure_rate}%) probes failed"
   fr: "%{failure_count} sur %{total_probes} (%{failure_rate}%) sondes ont échoué"
   tr: "%{failure_count} / %{total_probes} (%{failure_rate}%) sondajın başarısız olması"
   it: "%{failure_count} di %{total_probes} (%{failure_rate}%) prove fallite"
   pt: "%{failure_count} de %{total_probes} (%{failure_rate}%) sondas falharam"
   zh: "%{failure_count} 个失败，共探测到 %{total_probes} 个（%{failure_rate}%）"
+  ar: "فشلت %{failure_count} من %{total_probes} (%{failure_rate}%) الاختبارات"
 status_failed:
   en: "Failed"
   fr: "Échec"
   tr: "Başarısız"
   it: "Fallito"
   pt: "Falhou"
   zh: "失败"
+  ar: "فشل"
 status_running:
   en: "Running"
   fr: "En cours"
   tr: "Çalışıyor"
   it: "In esecuzione"
   pt: "Executando"
   zh: "运行中"
+  ar: "جاري التشغيل"
 status_frozen:
   en: "Frozen"
   fr: "Gelé"
   tr: "Dondurulmuş"
   it: "Congelato"
   pt: "Congelado"
   zh: "冻结"
+  ar: "مجمد"
 awaiting_data:
   en: "Awaiting data..."
   fr: "En attente de données..."
   tr: "Veri bekleniyor..."
   it: "In attesa di dati..."
   pt: "Aguardando dados..."
   zh: "等待数据……"
+  ar: "في انتظار البيانات..."
 header_help:
   en: "help"
   fr: "aide"
   tr: "yardım"
   it: "aiuto"
   pt: "ajuda"
   zh: "帮助"
+  ar: "مساعدة"
 header_settings:
   en: "settings"
   fr: "paramètres"
   tr: "ayarlar"
   it: "impostazioni"
   pt: "configurações"
   zh: "设置"
+  ar: "الإعدادات"
 header_quit:
   en: "quit"
   fr: "quitter"
   tr: "cıkış"
   it: "uscita"
   pt: "sair"
   zh: "退出"
+  ar: "إنهاء"
 title_hops:
   en: "Hops"
   fr: "Sauts"
   tr: "Atlananlar"
   it: "Salti"
   pt: "Saltos"
   zh: "跳"
+  ar: "قفزات"
 title_frequency:
   en: "Frequency"
   fr: "Fréquence"
   tr: "Sıklık"
   it: "Frequenza"
   pt: "Frequência"
   zh: "频率"
+  ar: "تردد"
 title_samples:
   en: "Samples"
   fr: "Échantillons"
   tr: "Örnekler"
   it: "Campioni"
   pt: "Amostras"
   zh: "样本"
+  ar: "عينات"
 title_traces:
   en: "Traces"
   fr: "Traces"
   tr: "İzler"
   it: "Tracce"
   pt: "Rastreios"
   zh: "跟踪"
+  ar: "التتبعات"
 title_flows:
   en: "Flows"
   fr: "Flux"
   tr: "Akışlar"
   it: "Flussi"
   pt: "Fluxos"
   zh: "流量"
+  ar: "تدفقات"
 title_map:
   en: "Map"
   fr: "Carte"
   tr: "Harita"
   it: "Mappa"
   pt: "Mapa"
   zh: "地图"
+  ar: "خريطة"
 title_help:
   en: "Help"
   fr: "Aide"
   tr: "Yardım"
   it: "Aiuto"
   pt: "Ajuda"
   zh: "帮助"
+  ar: "مساعدة"
 title_settings:
   en: "Settings"
   fr: "Paramètres"
   tr: "Ayarlar"
   it: "Impostazioni"
   pt: "Configurações"
   zh: "设置"
+  ar: "الإعدادات"
 bsod_failed:
   en: "Trippy Failed :("
   fr: "Trippy a échoué :("
   tr: "Trippy Başarısız :("
   it: "Trippy ha avuto un problema :("
   pt: "Trippy falhou :("
   zh: "Trippy 失败 :("
+  ar: "فشل Trippy :("
```

Comment:
Google translate tells me that "فشل متوقع" means "Expected failure" which seems wrong?  Also, can you show me how it would look in the full context (i.e. is it "Trippy فشل متوقع :(")

Reply:
It is very new word and has different meaning 
An adjective meaning cool, freaky, groovy, amazing, or all of the above, depending on the context in which it's used. Although "trippy" is a fairly new word, the root word, "trip," refers to soft psychedelic trips and has been in use since the 60's.

Person 1: "Whoa, dude. Those yellow fractals in the air are pretty trippy..."

Person 2: "Um, I think you've seriously gotta cut the acid. That's the McDonald's sign."

Courtesy: Urban Dictionary: trippy